src/effects.js

```diff
@@ -78,7 +78,6 @@ function createTween( value, prop, animation ) {
 }
 
 function defaultPrefilter( elem, props, opts ) {
-	/* jshint validthis: true */
```

Comment:
I'm fine with adding it later once we resolve the issues with it. Just maybe create a ticket about that so that we don't forget.